tasks/configure-cli.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Set PHP CLI memory limit.
+  block:
+    - name: Ensure PHP CLI config directory exists.
+      file:
+        dest: "/etc/php/{{ php_default_version_debian }}/cli/conf.d/"
```

Comment:
I'd still rather not merge it if it only works on Debian.

Reply:
@geerlingguy But if tests are passing, doesn't this prove that it also works on Red Hat, or whatever?

Reply:
well
- you're adding a feature that'll be only compatible with 1 out of X OS options
- tests are not that extensive, you could make a test that shows it doesn't work on RHEL distros and fix it accordingly

Reply:
The role still works, but if I add a feature that has a default variable that only applies to one distro (or a few) of any that are supported, then I start having to remember (for myself) which features of the role work on only one distro but not the other, and I have to start adding complexity to the documentation for features which shouldn't need to be distro-specific.

I prefer for my own roles to have defaults that work across any distro the role supports, but I'm always happy for people to fork my roles and modify them according to their own needs. I typically only add features that work on all distros unless it's something extremely specific to one OS (e.g. apt PPA url settings).

Reply:
Fair points.  To benefit everyone, however, can we tag this as Planned and then let others push to the PR to complete it eventually, when someone else gets to it?  GitLab allows this, but I don't work on GitHub enough to know if that's possible here.

Unless someone is able to provide funding (please [contact us](https://consensus.enterprises/#contact) if interested), I have no plans to add Red Hat support here, but it would be a shame to simply throw away the work I did.